vocode/streaming/models/synthesizer.py

```diff
@@ -226,3 +227,16 @@ class PollySynthesizerConfig(SynthesizerConfig, type=SynthesizerType.POLLY.value
     language_code: str = DEFAULT_POLLY_LANGUAGE_CODE
     voice_id: str = DEFAULT_POLLY_VOICE_ID
     sampling_rate: int = DEFAULT_POLLY_SAMPLING_RATE
+
+
+DEFAULT_CARTESIA_MODEL_ID = 'upbeat-moon'
+DEFAULT_CARTESIA_VOICE_ID = '5345cf08-6f37-424d-a5d9-8ae1101b9377'
+DEFAULT_CARTESIA_OUTPUT_FORMAT = 'pcm'
+DEFAULT_CARTESIA_SAMPLING_RATE = SamplingRate.RATE_44100
+
+
+class CartesiaSynthesizerConfig(SynthesizerConfig, type=SynthesizerType.CARTESIA.value):  # type: ignore
+    model_id: str = DEFAULT_CARTESIA_MODEL_ID
+    voice_id: str = DEFAULT_CARTESIA_VOICE_ID
+    output_format: str = DEFAULT_CARTESIA_OUTPUT_FORMAT
```

Comment:
we should use the `SynthesizerConfig` default of `audio_encoding` for defining audio encodings and set it to the string expected by the cartesia API similar to [here](https://github.com/ajar98/vocode-python/blob/main/vocode/streaming/synthesizer/play_ht_synthesizer_v2.py#L52-L55)